tests/devices/qutrit_mixed/test_qutrit_mixed_apply_operation.py

```diff
@@ -31,6 +31,24 @@
 ]
 
 subspaces = [(0, 1), (0, 2), (1, 2)]
+kraus_matrix = np.array([[0, 0, 1], [1, 0, 0], [0, 1, 0]], dtype=complex)
+
+
+class CustomChannel(Channel):  # pylint: disable=too-few-public-methods
+    num_params = 1
+    num_wires = 1
+
+    def __init__(self, p, wires, id=None):
+        super().__init__(p, wires=wires, id=id)
+
+    @staticmethod
+    def compute_kraus_matrices(p):
+        if math.get_interface(p) == "tensorflow":
+            p = math.cast_like(p, 1j)
+
+        K0 = math.sqrt(1 - p + math.eps) * math.convert_like(math.eye(3, dtype=complex), p)
```

Comment:
in the case p=1

Reply:
Oh I see, it's there to just prevent some numerical hiccups?

Reply:
Yeah, its not really necessary here though since I don't think I run into those issues with my selected values, but I added it just to be safe/consistent with other implementations.